advanced-cache.php

```diff
@@ -336,8 +336,12 @@ function do_variants($dimensions = false) {
                 if ( is_array($dimensions) ) {
                         ksort($dimensions);
                         foreach ( $dimensions as $key => $function ) {
-                                $fun = create_function('', $function);
-                                $value = $fun();
+
+                                $fun = function($function) {
```

Comment:
i feel we should not define the variable as this would allow calling $fun outside of this context. can we just
`$value = function( $function ) { return eval( $function ); }; `
instead to keep it anonymous.

Reply:
Yes, that should work just fine as well